web/src/components/historySection.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import loadable from '@loadable/component';
+import { slidesToShowPlugin } from '@brainhubeu/react-carousel';
+
+import HistoryItem from './historyItem';
+
+const Carousel = loadable(() => import('./carouselWrapper'));
+
+// const isBrowser = typeof window !== 'undefined';
+
+export default function History({ years }) {
+    const [activeTab, setActiveTab] = useState(years[0].id);
+    const [buttonComponentLogic, setButtonComponentLogic] = useState(false);
+
+    const setHistoryTimeline = () => {
+        if (window.innerWidth < 600) {
+            setButtonComponentLogic(true);
+        }
+    };
+
+    useEffect(() => {
+        setHistoryTimeline();
+        window.addEventListener('resize', setHistoryTimeline);
+
+        return () => {
+            window.removeEventListener('resize', setHistoryTimeline);
+        };
+    }, []);
+
+    return (
+        <section className="container my-24">
+            <h2 className="uppercase text-ts-h2 mb-4">History</h2>
+            <div>
+                <div className="flex gap-20">
+                    {buttonComponentLogic ? (
+                        <Carousel
+                            plugins={[
+                                'fastSwipe',
+                                {
+                                    resolve: slidesToShowPlugin,
+                                    options: {
+                                        numberOfSlides: 5,
+                                    },
+                                },
+                            ]}
+                        >
+                            {years.map((year) => {
+                                return (
+                                    <button
+                                        type="button"
+                                        onClick={() => setActiveTab(year.id)}
+                                        key={year.id}
+                                        isActive={activeTab === year.id}
+                                        className={
+                                            year.id === activeTab
+                                                ? 'underline font-bold'
+                                                : ''
+                                        }
+                                    >
+                                        {year.year}
+                                    </button>
+                                );
+                            })}
+                        </Carousel>
+                    ) : (
+                        <>
```

Comment:
Do you need the fragment here?

Reply:
@asasmith yeah

Reply:
I was able to remove it without issue.

Reply:
<img width="624" alt="Screen Shot 2021-06-14 at 5 33 19 PM" src="https://user-images.githubusercontent.com/61708936/121962332-a11ea400-cd36-11eb-90a2-925806cfbd72.png">

Reply:
ERROR in 
/Users/kaitlynbreuil/Desktop/airtec/web/sr
c/components/historySection.js
  67:30  error  Parsing error: Unexpected
token, expected "," (67:30)

Reply:
Try removing the `{}` from your `map()`